pkg/apis/ignite/v1alpha2/types.go

```diff
@@ -52,16 +52,10 @@ type OCIImageClaim struct {
 // OCIImageSource specifies how the OCI image was imported.
 // It is the status variant of OCIImageClaim
 type OCIImageSource struct {
-	// ID defines the source's ID (e.g. the Docker image ID)
-	ID string `json:"id"`
+	// ID defines the source's content ID (e.g. the canonical OCI path or Docker image ID)
+	ID *meta.OCIContentID `json:"id"`
```

Comment:
add omitempty?

Reply:
Same as above, IMO this field is required.